samples/XAMLCatalog/XAMLCatalog/UIKitControls/UIButtonWithControlsViewController.m

```diff
@@ -95,7 +95,7 @@ - (TestEnabledUITextField*)createTestEnabledUITextField {
 }
 
 - (void)createButtonWithTypeControls {
-    NSArray* segmentButtonTypeContent = [NSArray arrayWithObjects:@"Default", @"UIButtonTypeCustom", @"UIButtonTypeSystem", nil];
+    NSArray* segmentButtonTypeContent = [NSArray arrayWithObjects:@"Default", @"Custom", @"System", nil];
```

Comment:
Yes that's a good idea. I'll add that as a fourth button even though it's just system under the covers.

---
In reply to: [103040445](https://github.com/Microsoft/WinObjC/pull/2093#discussion_r103040445) [](ancestors = 103040445)